templates/takeovers/_juju-webinar-takeover.html

```diff
@@ -0,0 +1,12 @@
+<div lang="en" data-lang="all" class="p-strip--image is-dark js-takeover {% if not default %}u-hide{% endif %}" style="background-image:url('{{ ASSET_SERVER_URL }}30c32559-juju-k8s-bundle-folds.png'), linear-gradient(-121deg, #022935 0%, #044F66 57%, #067597 100%);">
+    <div class="row u-vertically-center">
+        <div class="col-6">
+            <h1 style="font-weight:100;">Keep up with evolving software</h1>
+            <h4>Learn how Juju can help solve software management complexity.</h4>
+            <p><a href="" class="p-button--neutral">Read more</a></p>
```

Comment:
@anthonydillon thank you for that. It has been added now. Though I am not sure why removing the margin at the top would add space not remove it on the small screens though, since p defaults to no margin top.